config/routes.rb

```diff
@@ -4,6 +4,11 @@
   devise_for :admins, path: 'admins'
   devise_for :employees, path: 'employees'
   resources :kudos
-    get '/admin' => "pages#dashboard", :as => :admin_root
+  namespace :admins do
+    resources :kudos, only: [:index, :destroy]
+    resources :employees, only: [:index, :edit, :update, :destroy]
+    resources :company_values
+  end
+  get '/admin' => "admins/pages#dashboard", :as => :admin_root
```

Comment:
Nitpick: You can also use `root "paged#dashboard"` under namespace :admins instead of this line to get generated path

Reply:
Thank you for your insight Szymon. I've implemented all of your advice except for this one as it simply doesn't work. When i did as you suggested the button 'Admin panel' got me to the kudos#index which is the 'prime' root. But nervertheless thanks again for all your comments and I hope to see you again at my future PR's :)

Reply:
No problem, make sure that the route names in links are correct after change. If I recall corectly it'll be "admins_root" and not "admin_root", you might also need to add "to:" before route like this:
```route to: "pages#dashboard"
.....
route to: "kudos#index"```

[Here's rails guides article about it](https://guides.rubyonrails.org/routing.html#using-root)